src/EnergyPlus/OutputProcessor.cc

```diff
@@ -5871,699 +4317,164 @@ void SetInitialMeterReportingAndOutputNames(EnergyPlusData &state,
     using namespace OutputProcessor;
 
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-    int indexGroupKey;
     std::string indexGroup;
     auto &op = state.dataOutputProcessor;
 
-    if ((FrequencyIndicator == ReportingFrequency::EachCall) ||
```

Comment:
Can simplify this function because value tracking for different reporting frequencies is now done using an array.

Reply:
Sheesh. :+1: